docfiles/docs.js

```diff
@@ -50,14 +50,14 @@ function dirAuto($el) {
 }
 
 function searchSubmit(form) {
-    if (pxt && pxt.tickEvent) pxt.tickEvent("docs.search", { 'source': form.id })
+    if (pxt && pxt.tickEvent) pxt.tickEvent("docs.search", { 'source': form.id }, /*interactiveConsent*/true)
```

Comment:
this notation is going to hard to enfore in the future. Instead, you could define an Options interface 
```
interface Options {
    interactiveConsent?: boolean;
}
```
and do
```
tickEvent(...., { interactiveConsent: true })
```
that way, TypeScript forces us to explain the meaning of that boolean value.

Reply:
Sure